modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/GetEntryRenderDataMVCResourceCommand.java

```diff
@@ -522,6 +535,45 @@ else if (ctEntry.getChangeType() !=
 			}
 		}
 
+		String workflowView = null;
+		Map<String, String> workflowData = new LinkedHashMap<>();
+
+		CTSQLModeThreadLocal.CTSQLMode ctSQLMode =
+			_ctDisplayRendererRegistry.getCTSQLMode(
+				ctCollection.getCtCollectionId(), ctEntry);
+
+		T model = _ctDisplayRendererRegistry.fetchCTModel(
+			ctCollection.getCtCollectionId(), ctSQLMode,
+			ctEntry.getModelClassNameId(), ctEntry.getModelClassPK());
+
+		if (model instanceof WorkflowedModel) {
+			long groupId = 0;
+
+			if (model instanceof GroupedModel) {
+				GroupedModel groupedModel = (GroupedModel)model;
+
+				groupId = groupedModel.getGroupId();
+			}
+
+			if (groupId != 0) {
+				boolean workflowEnabled =
+					_workflowDefinitionLinkLocalService.
+						hasWorkflowDefinitionLink(
+							ctEntry.getCompanyId(), groupId,
+							_portal.getClassName(
+								ctEntry.getModelClassNameId()));
```

Comment:
Maybe add this to the `CTDisplayRendererRegistryImpl` so that we can easily check if a ctEntry has workflow enabled.